lib/signet/vm.ex

```diff
@@ -49,7 +57,9 @@ defmodule Signet.VM do
       :memory,
       :tstorage,
       :reverted,
-      :return_data
+      # TODO: Should return data be a stack
```

Comment:
Nesting calls could technically overwrite each other, but since we aren't supporting calls except to FFIs, this should probably be okay.